### PR TITLE
Githubbin

### PR DESCRIPTION
**Pull request by simgu**
Only good code!

### CODE REVIEW

**Comment by simgu** [comment]
Solved some nice conflicts

**Comment by simgu** [comment]
:bowtie:

**Comment by simgu** [comment]
:trollface::trollface::trollface::trollface::trollface::trollface::trollface::trollface::trollface::trollface::trollface::trollface::trollface::trollface::trollface::trollface::trollface::trollface::trollface::trollface::trollface::trollface::trollface::trollface::trollface::trollface::trollface::trollface::trollface::trollface::trollface::trollface::trollface::trollface::trollface::trollface::trollface::trollface:

**Comment by simgu** [comment]
Less caching now :snake:

File: src/Web/Controllers/HomeController.cs [hunk 1]
```diff
@@ -34,7 +34,7 @@ public IActionResult Privacy()
             return View();
         }
 
-        [ResponseCache(Duration = int.MaxValue, Location = ResponseCacheLocation.None, NoStore = true)]
+        [ResponseCache(Duration = 100, Location = ResponseCacheLocation.None, NoStore = true)]
```
Comment (elindanielsson) [inline src/Web/Controllers/HomeController.cs]: Much better, thank you!

**Comment by simgu** [comment]
Thank you!

**Pull request merged**